src/components/LessonInfoDisplay/LessonInfoDisplay.js

```diff
@@ -1,13 +1,40 @@
 import React, { useState } from "react";
+import { Col, Row } from "reactstrap";
 import { compose } from "recompose";
 import { withRouter, Redirect } from "react-router-dom";
+import { useRef } from "react";
+import useOutsideClick from "../WordSelector/useOutsideClick";
 import "./LessonInfoDisplay.scss";
 
-function LessonInfoDisplay(props) {
+const LessonInfoDisplay = props => {
```

Comment:
why did you make this a const function with this syntax, what difference is this making?